plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/cwc/controller/chat/telemetry/TelemetryHelper.kt

```diff
@@ -410,12 +414,22 @@ class TelemetryHelper(private val project: Project, private val sessionStorage:
     companion object {
         private val logger = getLogger<TelemetryHelper>()
 
-        fun recordOpenChat() {
+        private fun getQConnection(project: Project): ToolkitConnection? = ToolkitConnectionManager.getInstance(
+            project
+        ).activeConnectionForFeature(QConnection.getInstance())
+
+        fun recordOpenChat(project: Project) {
             Telemetry.amazonq.openChat.use { it.passive(true) }
+            if (getQConnection(project) == null) {
+                AuthTelemetry.signInPageOpened()
+            }
```

Comment:
should this be
```suggestion
            if (getQConnection(project) == null) {
                AuthTelemetry.signInPageOpened()
            } else {
                Telemetry.amazonq.openChat.use { it.passive(true) }
            }
```

Reply:
it looks like vsc emits these separately. openChat is ambiguously: "When user opens CWSPR chat panel". 

separating these metrics *would* make open/close chat a little more meaningful.